lib/membrane/aac/parser/esds.ex

```diff
@@ -45,7 +45,26 @@ defmodule Membrane.AAC.Parser.Esds do
     stream_priority = 0
 
     {section_3, <<>>} = unpack_esds_section(esds, 3)
-    <<_elementary_stream_id::16, ^stream_priority, rest::binary>> = section_3
+
+    <<_elementary_stream_id::16, stream_dependence_flag::1, url_flag::1, ocr_stream_flag::1,
+      _stream_priority::5,
+      rest::binary>> =
+      section_3
+
+    rest =
+      Bunch.then_if(rest, stream_dependence_flag != 0, fn binary ->
```

Comment:
nit: `then_if` looks best when used in pipe:
```suggestion
      rest
      |> Bunch.then_if(stream_dependence_flag != 0, fn binary ->
```